src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -3098,6 +3100,17 @@ public int getWidth()
             return width;
         }
 
+        /**
+         * Whether or not this attachment is from an ephemeral Message.
+         * <br>If this Attachment is ephemeral, it will automatically be removed after 2 weeks. The attachment guaranteed to be available as long as the message itself exists.
```

Comment:
```suggestion
         * <br>If this Attachment is ephemeral, it will automatically be removed after 2 weeks. The attachment is guaranteed to be available as long as the message itself exists.
```